packages/trader/src/Modules/Account/Sections/Profile/PersonalDetails/personal-details.jsx

```diff
@@ -25,6 +25,7 @@ import FormSubmitErrorMessage                  from '../../ErrorMessages/FormSub
 import LoadErrorMessage                        from '../../ErrorMessages/LoadErrorMessage';
 import { LeaveConfirm }                        from '../../../Components/leave-confirm.jsx';
 import { FormFooter, FormBody, FormSubHeader } from '../../../Components/layout-components.jsx';
+import Icon from '../../../../../Assets/icon.jsx';
```

Comment:
this can just be `Assets/icon.jsx` instead of the relative path. and the `from` should be aligned with the `from` above

Reply:
Done ... thanks for review